thrift/src/test/java/com/linecorp/armeria/it/client/retry/RetryingRpcClientTest.java

```diff
@@ -116,4 +124,46 @@ public void execute_reachedMaxAttempts() throws Exception {
         assertThatThrownBy(() -> client.hello("hello")).isInstanceOf(Exception.class);
         verify(serviceHandler, times(1)).hello("hello");
     }
+
+    @Test
+    public void execute_void() throws Exception {
+        RetryStrategy<RpcRequest, RpcResponse> retryStrategy = new MaxTimeRetryStrategy<>(2);
+
+        DevNullService.Iface client = new ClientBuilder(server.uri(BINARY, "/thrift-devnull"))
+                .decorator(RpcRequest.class, RpcResponse.class,
+                           RetryingRpcClient.newDecorator(retryStrategy, 3)
```

Comment:
Could you reuse `ONLY_HANDLES_EXCEPTION` retry strategy than `MaxTimeRetryStrategy`?

Reply:
done